### DISCUSSION TITLE
Allow SettingsExpander to expand upwards

### DISCUSSION THREAD

**Discussion by qhy040404** [CommunityToolkit/Windows#440] [Ideas]
In the case where the upper part of the part is content and the lower part is settings, SettingsExpander that expands upwards will be more suitable.
Can we add ExpandDirection to SettingsExpander?

**Comment by niels9001** [comment]
Do you have a specific use case in mind? Or some sort of mock-up of what that could look like 😁? Just to figure out how this would look like
